epub33/core/index.html

```diff
@@ -3649,6 +3649,16 @@ <h5>Unicode Restrictions</h5>
 					<section id="sec-xhtml-deviations-discouraged" class="informative">
 						<h5>Discouraged Constructs</h5>
 
+						<section id="sec-xhtml-deviations-base">
+							<h6>The <code>base</code> Element</h6>
+
+							<p id="confreq-html-vocab-base">
+								The [[HTML]] <a data-cite="html#the-base-element"><code>base</code></a> element can be used to specify the <a data-cite="html#document-base-url">document base URL</a> for the purposes of parsing URLs.
+								When using it in an <a>EPUB Publication</a>, the interpretation of the <code>base</code> elements may inadvertently result in references to <a>Remote Resources</a>, i.e., resources that are outside the <a>EPUB Container</a>. The behavior of Reading Systems handling such resources may depend on whether they run as a separate application or as part of a Web Browser, and whether they are on line or off line. To avoid significant interoperability issues, EPUB Creators should not use the <code>base</code> element. 
```

Comment:
Generally okay with this except for this part:

"When using it in an EPUB Publication, the interpretation of the base elements may inadvertently result in references to Remote Resources, i.e., resources that are outside the EPUB Container."

The next sentence seems to be focused on web resources more so than resources on the file system outside the container, but both are potentially problematic depending on what the base element specifies.

It might help to add a specific example, so maybe more like:

"Using the `base` element in an EPUB Publication may cause Reading Systems to misinterpret the location of resources (e.g., relative links to other documents in the publication might appear as links to a web site if the `base` element specifies an absolute URL)."

Reply:
Yep, handled.